Dockerfile

```diff
@@ -18,6 +18,10 @@ COPY --from=build-env /root/build/sscd /usr/bin/
 
 RUN apk add gcompat bash curl
 
+RUN addgroup -S sscd && adduser -S sscd -G sscd
+USER sscd
+WORKDIR /home/sscd
```

Comment:
Wait, this is a breaking change and will require a migration. Is it worth? If so, we will need to update the guides and everywhere we are using docker. Basically, current validators running as root have all the files under `/root/.ssc`. If you change the home that directory will be empty.

Reply:
Yep, it's not worth to do now so I closed this PR, we have more than enough to do without it.